### PR TITLE
Remove status label in dataviewer

### LINKED ISSUE #314: Remove status label in dataviewer
### Feature you want to implement
I want to remove the status label in dataviewer, as its size keeps changing whenever the status is changed. It makes the plot image size inconsistent.

### How the feature is implemented
Remove the status label and leave a log instead.

### PR DESCRIPTION
**Pull request by kangz12345**
I think that the status message is not necessary and it is enough to use logging system.
Also, I removed `stopped` signal of `_RealtimeDatasetThread` as it seems unnecessary.

This closes #314.

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -8,4 +8,7 @@ min-similarity-lines=10
 argument-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
 attr-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
 method-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
-variable-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
\ No newline at end of file
+variable-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
+
+[DESIGN]
+max-positional-arguments=10
```
Comment (BECATRUE) [inline .pylintrc:14]: Why did you add this argument here?
This was already added in `develop` branch, hence does it cause a conflict error?

Comment (kangz12345) [reply .pylintrc:14]: I think that commit is added as I merged the `kangz12345/318/update-pylintrc` branch into this branch. Other PRs that I lately merged also have that commits and there was no conflict!

**Review by BECATRUE — APPROVED** [review]
